pulsar-client-cpp/lib/MultiTopicsConsumerImpl.h

```diff
@@ -99,7 +99,7 @@ class MultiTopicsConsumerImpl : public ConsumerImplBase,
     std::map<std::string, int> topicsPartitions_;
     mutable std::mutex mutex_;
     std::mutex pendingReceiveMutex_;
-    MultiTopicsConsumerState state_ = Pending;
+    std::atomic<MultiTopicsConsumerState> state_ = {Pending};
```

Comment:
```suggestion
    std::atomic<MultiTopicsConsumerState> state_{Pending};
```

`=` is not needed in brace initialization.